src/statistical/unigram.jl

```diff
@@ -3,57 +3,57 @@ struct SentencePieceModel
   vocab_map::Dict{String, Tuple{Float64, Int}}
   unk_id::Int
 end
-structure, To hold unknown token index and map of vocabulary to log probability and index   
+structure, To hold unknown token index and map of vocabulary to log probability and index
 """
 struct SentencePieceModel
     vocab_map::Dict{String, Tuple{Float64, Int}}
     unk_id::Int
 end
 
 """
-    load(ty::Type{T}, filenum::Int=1; unk_token="<unk>") where T<:PretrainedTokenizer
+    load_sp(ty::Type{T}, filenum::Int=1; unk_token="<unk>") where T<:PretrainedTokenizer
 use to initialize the `SentencePieceModel` by loading the file from `DataDeps`
 # Example
 ```julia-repl
 julia> spm = load(ALBERT_V1)
 ```
 """
-function load(ty::Type{T}, filenum::Int=1; unk_token="<unk>") where T<:PretrainedTokenizer
+function load_sp(ty::Type{T}, filenum::Int=1; unk_token="<unk>") where T<:PretrainedTokenizer
     filepath = @datadep_str tokenizer_files(ty)[filenum]
     name = tokenizer_files(ty)[filenum]
     filepath = "$filepath/$name"
-    load(filepath, unk_token=unk_token)  
+    load_sp(filepath, unk_token=unk_token)
 end
 
 """
-    load(path; unk_token="<unk>") 
+    load_sp(path; unk_token="<unk>")
 use to initialize the SentencePieceModel by providing `vocab filepath`
-"""    
-function load(path; unk_token="<unk>")
+"""
+function load_sp(path; unk_token="<unk>")
```

Comment:
Not sure about the rationale of this this change. Why is this needed? @oxinabox or @Ayushk4  should take a look here.

Reply:
The load function in GPT2 tokenizer was overriding this function. So, I changed it to separate functions that can be called by main **load** metthod. I'm not sure whether this way is optimized to performance or not.

Reply:
I think, It is fine to use `load_sp` (better call it `load_spu` (sentencepiece unigram)) and `load_gpt2` so that we can call it from the main `load`